components/modal/index.zh-CN.md

```
@@ -84,7 +84,7 @@ demo:
 | onOk | 点击确定回调 | function(e) | - |  | × |
 | afterOpenChange | 打开和关闭 Modal 时动画结束后的回调 | (open: boolean) => void | - | 5.4.0 | × |
 
-#### 注意
+#### 注意 {#note}
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)